engine/pkg/externalresource/broker/file_manager.go

```diff
@@ -163,7 +174,7 @@ func (m *LocalFileManager) RemoveResource(creator libModel.WorkerID, resName res
 	}
 
 	// Note that the resourcePath is actually a directory.
-	if err := os.RemoveAll(resourcePath); err != nil {
+	if err := os.RemoveAll(filePath); err != nil {
 		return derrors.ErrRemovingLocalResource.Wrap(err)
```

Comment:
Our local file resource is a file or a directory?
I think `resourcePath` is a more broad concept which we can have diferent implement. 
Why we change the name?

Reply:
- Our resource is a directory. This design choice is made so that the `brStorage` can be easily reused.
- `filePath` is for `the path of the resource directory in the file system`.

Reply:
Sounds a little ambiguity for `filePath` here. 
Pls modify the comment for it. `Note that the resourcePath is actually a directory.`

rest LGTM.